components/micro-gateway-core/src/main/ballerina/Ballerina.toml

```diff
@@ -9,4 +9,4 @@ target = "java8"
     path = "../../../target/org.wso2.micro.gateway.core-3.1.0-SNAPSHOT.jar"
 
 [dependencies]
-"wso2/jms" = "0.6.3"
\ No newline at end of file
+"wso2/jms" = "0.7.0"
```

Comment:
Better to add new line at the EOF

Reply:
added via https://github.com/wso2/product-microgateway/pull/939/commits/bf45068358e281152bcabdbd91db90a8a9fd9a94